addons/sourcemod/scripting/MyJailbreak/ratio.sp

```diff
@@ -1026,6 +1026,11 @@ void Menu_GuardQuestions(int client)
 
 		InfoPanel.Send(client, Handler_GuardQuestions, 20);
 	}
+	
+	if (hKeyValues)
```

Comment:
You don't need to check if handle is valid. That's just required with CloseHandle and was "removed" with delete.

Reply:
just ```delete hKeyValues;```?

Reply:
no just the if check, keep delete.

Reply:
https://github.com/shanapu/MyJailbreak/pull/283/commits/e89d76230afe2bb9a5692d71337e88ffeeec5f54
Same?